boards/QUARTER_BOARD/board.h

```diff
@@ -0,0 +1,771 @@
+/*
+    ChibiOS - Copyright (C) 2006..2015 Giovanni Di Sirio
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+#ifndef _BOARD_H_
+#define _BOARD_H_

```

Comment:
Just to note (so you know cool thing): C11 added `#pragma once` which does the same thing as:
```c
#ifndef _BOARD_H_
#define _BOARD_H_
...
#endif /* _BOARD_H_ */
```

Don't use it yet (since support for C11 is not that great in all compilers still), but I think it's good to know.